scripts/cmake/vcpkg_configure_cmake.cmake

```
@@ -319,15 +319,23 @@ function(vcpkg_configure_cmake)
         file(MAKE_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/vcpkg-parallel-configure)
         file(WRITE ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/vcpkg-parallel-configure/build.ninja "${_contents}")
 
-        message(STATUS "Configuring ${TARGET_TRIPLET}")
+        if(arg_Z_GET_CMAKE_VARS_USAGE)
+            message(STATUS "Getting CMake variables for ${TARGET_TRIPLET}")
+        else()
+            message(STATUS "Configuring ${TARGET_TRIPLET}")
+        endif()
         vcpkg_execute_required_process(
             COMMAND ninja -v
             WORKING_DIRECTORY ${CURRENT_BUILDTREES_DIR}/${TARGET_TRIPLET}-rel/vcpkg-parallel-configure
             LOGNAME ${arg_LOGNAME}
         )
     else()
         if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "debug")
-            message(STATUS "Configuring ${TARGET_TRIPLET}-dbg")
+            if(arg_Z_GET_CMAKE_VARS_USAGE)
+                message(STATUS "Getting CMake variables for ${TARGET_TRIPLET}")
+            else()
+                message(STATUS "Configuring ${TARGET_TRIPLET}-rel")
```

```suggestion
                message(STATUS "Configuring ${TARGET_TRIPLET}-dbg")
```

That is why there should be an abstraction for the build types.... then this silly copy&paste errors wouldn't happen